### PR TITLE
Add remote schemas directory

### PR DESCRIPTION
**Pull request by catalinpit**

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -185,9 +221,9 @@
 	  "short_desc": "GraphQL Mesh lets you use GraphQL to access data in remote APIs. Learn how to create a Data Graph with GraphQL Mesh & Hasura Remote Joins.",
 	  "url_path": "graphql-mesh-api",
 	  "github_repo": {
-	    "name": "praveenweb/graphql-mesh",
-	    "branch": "master",
-	    "readme_repo_path": "/README.md"
+	    "name": "hasura/graphql-mesh",
```
Comment (praveenweb) [inline config.json:224]: This needs to be `hasura/data-hub`

File: config.json [hunk 2]
```diff
@@ -5,13 +5,13 @@
...
-	    "branch": "master",
-	    "readme_repo_path": "/README.md"
+	    "name": "hasura/data-hub",
+	    "branch": "main",
+	    "readme_repo_path": "/remote-schemas/contentful/README.md"
 	  },
 	  "developed_by": "Contentful",
-	  "released_on": "June 10, 2021",
-	  "last_updated": "June 10, 2021",
+	  "released_on": "February 03, 2022",
```
Comment (praveenweb) [inline config.json:13]: Let us not change the original release date.

File: config.json [hunk 3]
```diff
@@ -77,13 +77,13 @@
...
-	    "name": "motleydev/graphcms-remote-schema",
+	    "name": "hasura/data-hub",
 	    "branch": "main",
-	    "readme_repo_path": "/README.md"
+	    "readme_repo_path": "/remote-schemas/graphcms/README.md"
 	  },
 	  "developed_by": "GraphCMS",
-	  "released_on": "June 10, 2021",
-	  "last_updated": "June 10, 2021",
+	  "released_on": "February 03, 2022",
```
Comment (praveenweb) [inline config.json:85]: Release date changed. Can keep the original for all instances.

Comment (catalinpit) [reply config.json:85]: Sure. Should I update only "last_updated"? Thank you.

Comment (praveenweb) [reply config.json:85]: Yes, last_updated can be changed.

Comment (catalinpit) [reply config.json]: Ok, done. Thank you!

**Pull request merged**